### PR TITLE
Add TSantos Serializer benchmark

### PR DESCRIPTION
**Pull request by tsantos84**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "symfony/property-access": "^4.0",
     "symfony/serializer": "^4.0",
     "thunderer/serializard": "^0.3",
-    "phpbench/phpbench": "^0.14.0"
+    "phpbench/phpbench": "^0.14.0",
+    "tsantos/serializer": "dev-jms-metadata-v2"
```
Comment (tsantos84) [inline composer.json:14]: The `dev-jms-metadata-v2` revision was intentionally as the `TSantos Serializer` still use `jms/metadata:1.x` version. So I've created a branch supporting `jms/metadata:2.x` to be able to add `TSantos Serializer` in this application

**Comment by goetas** [comment]
Very good results!

**Comment by tsantos84** [comment]
Yes! The hydrator approach increases substantially the performance.

File: composer.json (hunk 1 above)
Comment (goetas) [reply composer.json:14]: Are you planning a stable version using jms metadata v2?

Comment (tsantos84) [reply composer.json:14]: I've just released `TSantos Serializer` v4 fixed in PHP7.1 and JMS/Metadata requires PHP7.2. Not sure about release v5 right now.

Comment (tsantos84) [reply composer.json:14]: Is there a problem on fixing the package on a `dev-*` branch?

**Comment by goetas** [comment]
Thanks!

**Pull request merged**